### PR TITLE
Adds polyfills

### PR DESCRIPTION
**Pull request by BalintCsala**
This is aims to fix the different compatibility issues across browsers (mostly IE and egde). It solves the issue, where Edge doesn't clear the path and in some cases the background, when c.width is modified and replaces some Math function IE lacks (cosh, cbrt, trunc, fround, etc.)

### CODE REVIEW

File: dwitter/templates/dweet/dweet.html [hunk 1]
```diff
@@ -277,31 +278,10 @@
...
-          c._height = value;
-        },
-        get: function() {
-          return c._height;
-        },
-        configurable: true
+      $(c).resize(function() {
+        x.beginPath();
+        x.resetTransform();
+        x.clearRect(0, 0, obj.width, obj.height);
```
Comment (joeytwiddle) [inline dwitter/templates/dweet/dweet.html]: So `'resize'` gets triggered even if the canvas didn't change size? wtfdom!

**Comment by lionleaf** [comment]
Great! On a Mac atm, do most dweets work correctly in IE and Edge after this? And any idea of potential performance implications?

File: dwitter/templates/dweet/dweet.html (hunk 1 above)
Comment (joeysino) [reply dwitter/templates/dweet/dweet.html]: Mmm yeah, regarding @lionleaf's performance question, will this `clearRect` run on all platforms, whenever `c.width` is assigned?

Won't that impact performance on platforms where it's not needed?

**Comment by BalintCsala** [comment]
@lionleaf It doesn't seem like it has any performance effects, the dweets have the same FPS on my local version and on dwitter.net

**Comment by BalintCsala** [comment]
Also, yes, this should solve most of the problems with edge and if we notice something else not being reset, then we can add that too easily

**Pull request merged**